src/aics-image-viewer/components/StyleProvider/index.tsx

```diff
@@ -65,6 +65,11 @@ const theme = {
         text: palette.medPurple,
       },
       secondary: {
+        bg: "transparent",
+        text: palette.medPurple,
+        outline: palette.medPurple,
+      },
+      tertiary: {
```

Comment:
I'd made a mistake when I set up these CSS variables and mixed up the secondary and tertiary button styles. Some of the line changes here fix that by renaming variables accordingly.

![image](https://github.com/allen-cell-animated/website-3d-cell-viewer/assets/30200665/0c0fba52-8c16-4026-9dfa-8b0be5331ee9)